addons/web/static/src/core/model_field_selector/model_field_selector.js

```diff
@@ -89,6 +89,14 @@ export class ModelFieldSelector extends Component {
         }
         const { isInvalid, modelsInfo, names } = await this.fieldService.loadPath(resModel, path);
         const result = { isInvalid: !!isInvalid, displayNames: [] };
+        if (!isInvalid) {
+            const lastName = names[names.length - 1];
```

Comment:
use .at(-1)

Reply:
do the 2 arrays have the same length ?
is it always the case?